src/service-override/files.ts

```diff
@@ -517,7 +517,7 @@ const userDataFileSystemProvider = new InMemoryFileSystemProvider()
 void userDataFileSystemProvider.mkdir(URI.from({ scheme: Schemas.vscodeUserData, path: '/User/' }))
 
 const providers: Record<string, IFileSystemProvider> = {
-  'extension-fs': extensionFileSystemProvider,
+  extension: extensionFileSystemProvider,
```

Comment:
Should we go back to `extension` here?

Reply:
Or rather, why are we going back to `extension`?

Reply:
Why did we change in the first place?

Reply:
If we use `extension`, we're conflicting with the input of the gallery, and because of this we'll see breadcrumbs for extension views:

<img width="560" alt="Screenshot 2023-11-21 at 23 59 23" src="https://github.com/CodinGame/monaco-vscode-api/assets/587016/994e8a70-dbc6-454b-97b2-e7ffd3da31df">

Reply:
well spotted, replaced by 3debbf8e0a97908c509eb6aaf64a23544455984f then